### ISSUE TITLE
[Name Tag Tweaks] Error on load

### ISSUE THREAD

**Issue by silvigon** [Serilum/.issue-tracker#216]
## **Information**
Minecraft version: 1.16.1
Forge version: 32.0.98
Environment: Singleplayer

Mod name: Name Tag Tweaks
Mod version: 1.16-1.1

## **Description**
The game displays an error screen upon loading, with the following message:
```
Name Tag Tweaks (nametagtweaks) has failed to load correctly
Method public void com.natamus.nametagtweaks.Main.registerCommands(net.minecraftforge.event.RegisterCommandsEvent) has @SubscribeEvent annotation, but takes an argument that is not a subtype of the base type interface
```

## **Crash report**
Technically not a crash, but: [latest.log](https://github.com/ricksouth/serilum-mc-mods/files/5015886/latest.log)

**Comment by silvigon** [comment]
Just had basically the same error with a different mod, so maybe it could be a problem in Forge?

**Comment by ricksouth** [comment]
It was indeed something Forge changed, but it has been fixed in the latest file for Minecraft 1.16.2. Thanks for opening the issue and the interest!
https://www.curseforge.com/minecraft/mc-mods/name-tag-tweaks/files
